receiver/activedirectorydsreceiver/metadata.yaml

```diff
@@ -29,7 +29,7 @@ attributes:
     description: The type of value sent.
     type: string
     enum:
-      - distingushed_names
+      - distinguished_names
```

Comment:
@pjanotti: As code owner, should this change include a changelog?

Reply:
Thanks for the ping @crobert-1 - yes, this deserves a changelog. I am also not seeing the corresponding changes to the generated code: I would expect `AttributeValueTypeDistingushedNames` to also be fixed.

Reply:
Can someone explain what it would look like? (Or provide it)
I'm not opposed to such things, but...

Reply:
Add a file named `.chloggen\fix-attribute-typo-on-activedirectorydsreceiver.yaml` to your change and set its content to:
```yaml
change_type: breaking

component: activedirectorydsreceiver

note: Fixed typo in the attribute `distingushed_names`, renaming it to `distinguished_names`.

issues: [37606]

subtext:

change_logs: [user]
```

Check the `.chloggen/TEMPLATE.yaml` if you want to see a comment describing each entry on this file.

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/8849ae89f6c59e9229d7952142c19bcd4f183e04/.chloggen/fix-attribute-typo-on-activedirectorydsreceiver.yaml